runtime/str.go

```diff
@@ -1199,6 +1292,17 @@ func isDigit(c byte) bool {
 	return '0' <= c && c <= '9'
 }
 
+func isLower(c byte) bool {
+	return 'a' <= c && c <= 'z'
```

Comment:
Can you replace the other identical comparisons in this file (e.g. in isAlpha and strTitle, toUpper) with a call to this function? Ditto for isUpper().